_example/main.go

```diff
@@ -16,6 +16,16 @@ func main() {
 }
 
 /*
++main - generate example
+
```

Comment:
Ah, you deliberately dropped it.Hm, seems ok, but you should document about the format in the gen.Generate doc.